presto-orc/src/main/java/com/facebook/presto/orc/Filters.java

```diff
@@ -494,6 +495,7 @@ public boolean isEquality()
             extends Filter
     {
         private final HashMap<SubfieldPath.PathElement, Filter> filters = new HashMap();
+        private HashMap<Long, Filter> longToFilter;
```

Comment:
This change doesn't seem to belong to this PR. Aside from that, variable type should be Map (interface), not HashMap (implementation).